fluent/src/context.js

```diff
@@ -113,8 +113,20 @@ export class MessageContext {
       if (id.startsWith("-")) {
         // Identifiers starting with a dash (-) define terms. Terms are private
         // and cannot be retrieved from MessageContext.
+        if (this._terms.has(id)) {
+          if (typeof console !== "undefined") {
```

Comment:
In which environments can we expect `console` to be undefined?

Reply:
I don't think there's any. Can we add it to eslint to not throw and assume `console` is available? :)

Reply:
I throws to protect against `console.logs` sprinkled across the code after some debugging. `// eslint-disable-line no-console` should do the trick.

Reply:
If `console` is available in all envs, then what's the purpose of the `if typeof` check? Can we remove it?